### PR TITLE
Added tick format function for axes

### PR DESCRIPTION
**Pull request by lorem--ipsum**
@chaosmail I'll merge this if you can't, I need it soon-ish for work (yeah, I'm using n3-charts at work :-) )

### CODE REVIEW

**Comment by chaosmail** [comment]
holy moly, I will give a look at this today

File: src/LineChart.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ module n3Charts {
       };
 
       // Trigger the create event
-      eventMgr.trigger('create');
+      eventMgr.trigger('create', scope.options);
```
Comment (chaosmail) [inline src/LineChart.ts]: shouldn't we better use the parsed `options` object here?

File: src/factories/series/Dot.ts [hunk 2]
```diff
@@ -35,10 +34,10 @@ module n3Charts.Factory.Series {
       dots.enter()
         .append('circle')
         .attr('class', this.type)
-        .call(this.eventMgr.datumEnter(series))
-        .call(this.eventMgr.datumOver(series))
-        .call(this.eventMgr.datumMove(series))
-        .call(this.eventMgr.datumLeave(series))
+        .call(this.eventMgr.datumEnter(series, this.options))
```
Comment (chaosmail) [inline src/factories/series/Dot.ts]: I wonder why one would need the options in the callback functions.

**Comment by chaosmail** [comment]
this looks great! I will just await your answer on the comments but i guess it is good to merge. Wait. No. You are using this at work!!!!!!!!!!!!! That's awesome man. Let's better correct all details then

File: src/LineChart.ts (hunk 1 above)
Comment (lorem--ipsum) [inline src/LineChart.ts]: Hmmm yeah, but then I'll need to store it somewhere to not parse it twice (the update occurs after and I don't want to wait until then to trigger the create event, obviously)

File: src/factories/series/Dot.ts (hunk 2 above)
Comment (lorem--ipsum) [inline src/factories/series/Dot.ts]: To format the tooltip using the axis' tickFormat.

File: src/LineChart.ts (hunk 1 above)
Comment (chaosmail) [inline src/LineChart.ts]: hmm, we need to parse it on every updated anyway

File: src/factories/series/Dot.ts (hunk 2 above)
Comment (chaosmail) [inline src/factories/series/Dot.ts]: yeah, true

File: src/LineChart.ts (hunk 1 above)
Comment (lorem--ipsum) [inline src/LineChart.ts]: HMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM that's true. ^^

**Comment by lorem--ipsum** [comment]
They see me mergin', they hatin'

**Pull request merged**